criu/kerndat.c

```diff
@@ -1478,6 +1565,37 @@ int kerndat_try_load_new(void)
 	return 0;
 }
 
+static int root_only_init(void)
+{
+	int ret = 0;
+
+	if (opts.uid)
+		return 0;
+
+	if (!ret && kerndat_loginuid()) {
```

Comment:
all kerndat changes have to be moved into a separate patch.

Reply:
All kerndat changes are now in their own patch. Thanks.